addons/google_calendar/utils/google_event.py

```diff
@@ -193,3 +193,12 @@ def _get_model(self, env):
         if all(not e.is_recurrence() for e in self):
             return env['calendar.event']
         raise TypeError("Mixing Google events and Google recurrences")
+
+    def get_meeting_url(self):
+        if not self.conferenceData:
+            return False
+        video_meeting = next(filter(lambda entryPoints: entryPoints['entryPointType'] == 'video', self.conferenceData['entryPoints']), None)
```

Comment:
Why did you choose to use `next` here instead of just getting the first item in your list with `self.conferenceData['entryPoints'][:1]`?

Reply:
Hello @leonardopavanrocha , 

`self.conferenceData['entryPoints'][:1]` would have been sufficient if Google Meet was the only provider but in the case where the conference is managed by an other provider (Jitsi for example), the first element does not correspond to the video's informations that I need here. That's why I used `filter()` and `next()`. 

Do you see any better solution?

Reply:
Oh, sorry. I meant getting the first result out of the filter. Something like `filter(lambda entryPoints: entryPoints['entryPointType'] == 'video', self.conferenceData['entryPoints'])[:1]`. Both work fine, I just wanted to know if there was a specific reason to use `next`. I feel like `next` is normally used when you want to iterate through the list.

Reply:
Oooh it looks like there was a misunderstanding here :sweat_smile: 

In fact, there is no particular reason to use `next()` here. And I agree with you that, even if it works correctly, `next()` might not be the right choice here because I don't browse the list.

It can be replaced by `list(filter(lambda entryPoints: entryPoints['entryPointType'] == 'video', self.conferenceData['entryPoints']))[0]` as `filter(lambda entryPoints: entryPoints['entryPointType'] == 'video', self.conferenceData['entryPoints'])[:1]` can't, unfortunately, be used as is. 

Does this sound better to you @leonardopavanrocha ?

Reply:
@thcl-odoo Yes, this seems better :smile:

Reply:
Just updated the PR according to this
Thanks for your review :slightly_smiling_face: